### DISCUSSION TITLE
日本新創推元宇宙痛覺模擬器，不只有感覺，連痛覺都完美複製！

### DISCUSSION THREAD

**Discussion by Rosancl** [1101-GS4719-C/Python-Community-GS4719-C#616] [General]
**元宇宙受傷，現實世界也會痛？**

日本新創公司 H2L發明了一種腕帶，讓用戶在虛擬世界中四處走動時，也能感受到疼痛和物體重量等感覺。這項產品可檢測人體肌肉彎曲，讓用戶在虛擬世界中的角色模仿身體運動，並透過電流刺激佩戴者，讓用戶能夠感受物體的存在和重量，甚至能感覺到皮膚被捏住或被鳥啄皮膚。

https://buzzorange.com/techorange/2022/04/01/feel-real-life-pain-in-metaverse/

**Comment by minniewei** [comment]
感覺未來，我們會更加分不清楚甚麼是現實甚麼是虛擬，畢竟連痛覺都可以被模擬。

**Comment by cordydeng** [comment]
這項技術不知道有什麼用途，不過感覺還蠻奇特的

**Comment by jcpan821** [comment]
說不定我們現在也在模擬器裡

**Comment by d26488399** [comment]
可能可以讓家長用來控制小孩的遊玩時間，例如時間到了就放一堆狗出來咬他，真的會痛小孩可能就不會繼續玩了。

**Comment by 11timyyy** [comment]
感覺很適合有特殊癖好的人使用?!

**Comment by henrywen547** [comment]
這讓我想到之前看的小說跟電影-一級玩家，但最後真的需要思考如何從虛擬回到現實

**Comment by Lai27330** [comment]
感覺會分不清楚虛擬和現實，在虛擬世界裡還是不太希望能感受到痛覺

**Comment by awkward-willy** [comment]
越來越有電影「一級玩家」的感覺了呢
或許在未來，真的能實現全人類帶著頭戴裝置、身穿特殊衣服的元宇宙場景

**Comment by HsuChen88** [comment]
好奇在虛擬中也會痛了 還會想再去嗎
